aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-7.md

```diff
@@ -75,7 +75,7 @@ Fill in the user information fields on the left using the following information:
 | **User Name** | Administrator |
 | **Password** | password123! |
 | **Confirm Password** | password123! |
-| **E-mail** | (any e-mail address will work) |
+| **E-mail** | (any email address will work) |
```

Comment:
@guardrex  you missed the - on uppercase cases it seems.

Reply:
It refers to a UI element, so I left those alone. There were a few cases of this.

Reply:
> refers to a UI element

Let me elaborate a hair ... I guessed, perhaps incorrectly, that the UI actually has it as "E-mail" with the dash. Therefore, I wanted to match that. Since that might not be a good guess, I'll check these UIs today when I review this and confirm. If the UI does have a dash tho, I think we should leave it dashed in the topic to match.

Reply:
Gotcha @guardrex. Although ideally, UI would also follow the same rules. 😄

Reply:
Yes, you're right that you should match whatever the UI says.

Reply:
I guess what we could do if I catch the UI with a dash is track down the appropriate party and see about an internal/external issue with them to update their UI. This only happened in a couple of cases, so it shouldn't take much time.

Reply:
@Rick-Anderson @mairaw We're good here. :+1: No external links were touched on this.

The "E-mail" versions left in the repo are ...

aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-7.md
https://docs.microsoft.com/en-us/aspnet/mvc/overview/older-versions/mvc-music-store/mvc-music-store-part-7
Refers to the ASP.NET Web Site Administration Tool ... This UI can only be seen in the Smithsonian in D.C. at this point ... in the *Ancient Technology Exhibits* of the museum ... right next to the typewriter and the pager. :smile:

aspnet/web-forms/overview/older-versions-security/membership/creating-user-accounts-cs.md
https://docs.microsoft.com/en-us/aspnet/web-forms/overview/older-versions-security/membership/creating-user-accounts-cs
Refers to the CreateUserWizard page of a Stone Age template ... apparently one would chisel their email address into the UI ... literally! :smile:

The next one is just the VB version of the previous topic :point_up:.

Last one refers to a heading in this topic: https://docs.microsoft.com/en-us/aspnet/whitepapers/aspnet4/overview
It links to this section: https://docs.microsoft.com/en-us/aspnet/whitepapers/aspnet4/overview#0.2__Toc253429282
I didn't mod it because the whole topic uses title casing.

Reply:
Thanks for looking into each one of those @guardrex!